test/sanity/run-tests-all-clouds.sh

```diff
@@ -37,12 +37,11 @@ if [ -v aadClientSecret ]; then
 	sed -i "s/resourceGroup-input/$resourceGroup/g" $AZURE_CREDENTIAL_FILE
 	sed -i "s/location-input/$location/g" $AZURE_CREDENTIAL_FILE
 
-	sudo ${GO_BIN_PATH} test -v ./test/sanity/...
+	test/sanity/run-test.sh
 else
 	if [ -v subscriptionId ]; then
 		echo "skip sanity test in CI env"
 	else
-		# run test in user mode
-		go test -v ./test/sanity/...
+		test/sanity/run-test.sh
```

Comment:
revert this code change, run `go test -v ./test/sanity/...` in local

Reply:
done.